kvbc/include/resources-manager/AdaptivePruningManager.hpp

```diff
@@ -80,7 +80,7 @@ class AdaptivePruningManager {
   uint32_t current_pruning_pace_;
   uint64_t current_batch_size_;
   concordMetrics::Component metricComponent;
-  concordMetrics::Component::Handle<concordMetrics::AtomicGauge> ticksPerSecondMetric, batchSizeMetric,
+  concordMetrics::Component::Handle<concordMetrics::AtomicGauge> blocksPerSecondMetric, batchSizeMetric,
```

Comment:
Does it make sense to leave `ticksPerSecondMetric` and add the `blocksPerSecondMetric` one? I guess it would depend if we need `ticksPerSecondMetric`.

Reply:
IMO ticks per second is misleading, because it's blocks. Besides the metric "intervalBetweenTicks" already exists.